### PR TITLE
Made Services Page

### PR DESCRIPTION
**Pull request by Abdullah-Waqas13341**
Made Form to Post Services
Adjusted Routes

### CODE REVIEW

File: sellers/views.py
Comment (rabbiasajjad-devsinc) [inline sellers/views.py:1]: Add error handling using try-except in case the form validation fails

Comment (rabbiasajjad-devsinc) [reply sellers/views.py:1]: You can put form validation in a separate method as well.

Comment (Abdullah-Waqas13341) [reply sellers/views.py:1]: ok

Comment (rabbiasajjad-devsinc) [inline sellers/views.py:1]: Follow PEP import guidelines for the best practices

https://peps.python.org/pep-0008/#:~:text=Imports,-Imports%20should%20usually&text=Imports%20are%20always%20put%20at,Standard%20library%20imports.

**Review by rabbiasajjad-devsinc — APPROVED** [review]

**Review by rabbiasajjad-devsinc — APPROVED** [review]

**Pull request merged**